src/report.rs

```diff
@@ -26,7 +26,12 @@ pub struct Report {
 
 impl Report {
   /// Create a new instance.
-  pub fn new(method: Method, explanation: String) -> Self {
+  pub fn new(
+    name: &str,
+    version: &str,
+    method: Method,
```

Comment:
derive Copy for Method pls

Reply:
To be honest, do we really even need the method anymore?